apps/backend/.env.example

```
@@ -0,0 +1,37 @@
+# Example backend environment file (do not commit secrets)
+SESSION_SECRET_KEY="CHANGE_ME_SECURE_RANDOM"
+
+# Database (Neon Postgres)
+# Either set DATABASE_URL to a single connection string and let the app derive sync/async URLs
+# DATABASE_URL="postgresql://user:password@host/db?sslmode=require"
+# Or set explicit URLs:
+SYNC_DATABASE_URL="postgresql+psycopg://user:password@host/db?sslmode=require"
+ASYNC_DATABASE_URL="postgresql+asyncpg://user:password@host/db?ssl=require"
+
+# AI provider (OpenAI recommended for cloud)
+LLM_PROVIDER="openai"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)